pkg/operator/api/api.go

```diff
@@ -117,7 +130,7 @@ func (a *apiServer) Run(ctx context.Context, sec security.Handler) error {
 		return err
 	}
 	err = lis.Close()
-	if err != nil {
+	if err != nil && !errors.Is(err, net.ErrClosed) {
```

Comment:
are there any other errors being swallowed here with this new check?

Reply:
Not sure I follow, we are only ignoring the case when the listener errors because it has been closed.